android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/RpcClient.java

```diff
@@ -85,6 +88,8 @@
     public static final int MGR_SYNC = 31;
 
     private LocalSocket mSocket;
+    private Socket mTcpmSocke;
```

Comment:
`mTcpmSocke` -> `mTcpmSocket`

Reply:
You mean mTcpSocket. Yes, I will change my typo.